crates/iota/src/iota_commands.rs

```diff
@@ -1192,42 +1192,32 @@ fn prompt_for_environment(
             print!(
                 "Creating config file [{wallet_conf_path:?}] with default (Testnet) full node server and ed25519 key scheme."
             );
-        } else {
-            print!(
-                "Config file [{wallet_conf_path:?}] doesn't exist! Do you want to connect to an IOTA full node server? [y/N]: "
-            );
+            return Ok(IotaEnv::testnet());
```

Comment:
Why return? Shouldn't we still allow choosing a network? I mean I guess you're taking "acccept defaults" to mean all defaults but this feels not so useful. 🤷‍♀️

Reply:
That's the same behavior as before and the point of the accept default, if you want to select a network you don't accept the default

Reply:
Another aspect that disturbs a bit is that the print seems badly placed here, because it's not doing the things it says it does, i.e. it's not creating a config file at all here. It seems this should be moved to the caller tbh, because here it's just returning the default testnet env.

And another thing is that we probably should return early in line 1189, so we get rid of nesting most of the code in this function.

Reply:
Problem with the print is that you would then also need to have the same checks in another place, not helping with the readability
Did the early return here https://github.com/iotaledger/iota/pull/8003/commits/1cc1796a9db91b9db5a2cc44c28b3e3c3d943cb5